packages/e2e-tests/tests/playwright-01_basic-panel.test.ts

```diff
@@ -92,7 +92,8 @@ test("playwright-01: Basic Test Suites panel functionality", async ({
   expect(await sections.nth(0).textContent()).toMatch(/test body/i);
 
   const steps = getTestCaseSteps(selectedRow);
-  await expect(steps).toHaveCount(231);
+  const stepCount = await steps.count()
```

Comment:
This change broke Lint. Keep an eye on CI (or just run `yarn lint` locally) to spot this. Else CI won't even run your tests.